content/en/docs/install-operate/source.md

```diff
@@ -508,54 +453,3 @@ sudo ./userspace/falco/falco -c ../falco.yaml -r ../rules/falco_rules.yaml
 ```
 
 By default, falco logs events to standard error.
-
-
-### Run regression tests
```

Comment:
These are also out of date. We now have a different repo for testing which is used in Falco plus a bunch of unit and integration tests. We can replace this info with an updated version but for now this is just misleading.